File: app/src/main/java/vgame/ir/data/local/entity/CollegeEntity.java
package vgame.ir.data.local.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.SerializedName;

@Entity(tableName = "college")
public class CollegeEntity {

    @PrimaryKey
    @SerializedName("id")
    private int id;

    @SerializedName("uni_id")
    private int uni_id;

    @SerializedName("name")
    private String name;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getUni_id() {
        return uni_id;
    }

    public void setUni_id(int uni_id) {
        this.uni_id = uni_id;
    }
}

File: app/src/main/java/vgame/ir/data/local/entity/CollegeEntity.kt
package vgame.ir.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

import com.google.gson.annotations.SerializedName

@Entity(tableName = "college")
class CollegeEntity {

    @PrimaryKey
    @SerializedName("id")
    var id: Int = 0

    @SerializedName("uni_id")
    var uni_id: Int = 0

    @SerializedName("name")
    var name: String? = null
}